src/js/gutenberg-plugins/content-resizing-plugin.js

```diff
@@ -142,6 +146,12 @@ const ContentResizingPlugin = () => {
 				await resizeContent();
 			} )();
 		}
+
+		if ( 'grow' === resizingType ) {
+			setModalTitle( __( 'Expanded text suggestions', 'classifai' ) );
+		} else {
+			setModalTitle( __( 'Condensed text suggestions', 'classifai' ) );
```

Comment:
This may be more complicated than what we care about but in the case where someone has it set to only display a single suggestion, `suggestions` doesn't make sense at that point. Very minor thing so may be fine to just leave

Reply:
Added here [6e9884d](https://github.com/10up/classifai/pull/774/commits/6e9884df745ba93cb4e464ede838dee64c95b934)